.github/workflows/examine-python-bindings.yml

```diff
@@ -19,7 +19,7 @@ on:
     paths:
       - ".github/workflows/examine-python-bindings.yml"
       - ".github/actions/**"
-      - "bindings/python/**"
+      - "bindings/python/native/**"
```

Comment:
Not really sure we should do that. It would mean we wouldn't trigger actions if we change the Python code itself. Also, it's inconsistent with the push section.

Reply:
Just added it because the workflow was running only for Python and I thought it's related to this, will undo